src/components/ScContract.js

```diff
@@ -60,70 +58,90 @@ function ScContract({ userAddress }) {
-              <option key={address} value={address}>
-                {`walletID #${index + 1} ${address}`}
-              </option>
-            ))}
-          </Form.Select>
-          <Button
-            disabled={!isAddress(scAddress) || factoryIsLoading}
-            variant="primary"
-            onClick={() => {
-              // console.log(addressIsReady);
-              setAddressIsReady(true);
-            }}
-          >
-            Display
-          </Button>{" "}
-          <Button
-            disabled={!addressIsReady || factoryIsLoading}
-            variant="danger"
-            onClick={() => {
-              // console.log(addressIsReady);
-              setAddressIsReady(false);
-            }}
-          >
-            Close Contract
-          </Button>
+      <div className="mt-5">
+        <div>
+          {factoryReadIsSuccess && (
+            <>
+              {" "}
+              <div className="factory-section">
+                <div className="factory-address">
+                  <span>Factory Address:</span>
+                  <span className="factory-address-value">
+                    {process.env.REACT_APP_FACTORY_ADDRESS}
+                  </span>
+                </div>
+                <div className="input-field">
+                  <Form.Select
+                    disabled={factoryIsLoading}
+                    aria-label="Default select example"
+                    className="mt-3 mb-2 factory-address-form"
+                    onChange={(e) => {
+                      if (isAddress(e.target.value)) {
+                        setScAddress(e.target.value);
+                      }
+                    }}
+                  >
+                    <option>Select Contract Address</option>
+                    {}
+                    {factoryReadData?.map((address, index) => (
+                      <option className="fs-5" key={address} value={address}>
+                        {`walletID #${index + 1} ${address}`}
+                      </option>
+                    ))}
+                  </Form.Select>
+                </div>
+                <div className="factory-button">
+                  <Button
+                    className="displayButton"
+                    disabled={!isAddress(scAddress) || factoryIsLoading}
+                    variant="danger"
+                    onClick={() => {
+                      // console.log(addressIsReady);
+                      setAddressIsReady(true);
+                    }}
+                  >
+                    Display
+                  </Button>{" "}
+                  <Button
+                    className="close-btn"
+                    disabled={!addressIsReady || factoryIsLoading}
+                    variant="danger"
+                    onClick={() => {
+                      // console.log(addressIsReady);
+                      setAddressIsReady(false);
+                    }}
+                  >
+                    Close Contract
```

Comment:
This button is not displaying properly. 
![Screenshot 2023-12-11 at 6 39 36 PM](https://github.com/warrenseah/multi-sig-wallet-frontend/assets/6447221/42bccc4d-4636-414b-a763-376b801d3e9f)